packages/widgets/src/components/NotificationCard/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IMapIcons } from '@ssa-ui-kit/core';
+
+export interface INotificationCardProps {
```

Comment:
Do we need **I** in the interfaces name? I think, we won't have intersection with NotificationCardProps

Reply:
done